packages/vue-searchbox/src/components/VueSearchbox.jsx

```diff
@@ -486,7 +507,8 @@ const VueSearchbox = {
                       promotedData,
                       customData,
                       resultStats,
-                      rawData
+                      rawData,
+                      querySuggestions: this.querySuggestionsList,
                     })}
                   {this.renderErrorComponent()}
                   {!renderAllSuggestions && isOpen && suggestionsList.length ? (
```

Comment:
@ShahAnuj2610 Add a task to update `renderAllSuggestions` to `render` as same as `vue-rs`.